templates/toolchains/index.html

```diff
@@ -15,301 +21,303 @@
 {% endblock body_class %}
 
 {% block content %}
-  <section class="p-section--hero">
-    <div class="row--50-50">
-      <div class="col">
-        <h1>Build apps with your favorite frameworks and toolchains on Ubuntu</h1>
-      </div>
-      <div class="col">
-        <div class="p-section--shallow">
-          <p>
-            <strong>Rapid releases, long term security and automation for efficient dev workflows.</strong>
-            <br>
-            Ubuntu offers native support for the most popular development framework and toolchains, with rapid release cycles, long term security support, compliance certification, minimal base containers, and additional automation for software builds and development environment setup.
-          </p>
-        </div>
-        <hr class="p-rule--muted">
-        <a href="https://documentation.ubuntu.com/ubuntu-for-developers/" class="p-button--positive">Ubuntu for developers docs</a>
-      </div>
-    </div>
-  </section>
+  {% call(slot) vf_hero(
+    title_text="Build apps with your favorite frameworks and toolchains on Ubuntu",
+    layout="50/50"
+  ) -%}
+  {%- if slot == 'description' -%}
+    <p>
+      <strong>Rapid releases, long term security and automation for efficient dev workflows.</strong>
+      <br />
+      Ubuntu offers native support for the most popular development framework and toolchains, with rapid release cycles, long term security support, compliance certification, minimal base containers, and additional automation for software builds and development environment setup.  
+    </p>
+  {%- endif -%}
+  {%- if slot == 'cta' -%}
+    <a href="https://documentation.ubuntu.com/ubuntu-for-developers" class="p-button--positive">Ubuntu for developers docs</a>
+  {%- endif -%}
+  {%- endcall -%}
 
   <section class="p-section">
-    <div class="u-fixed-width">
-      <hr class="p-rule">
-    </div>
-    <div class="row--50-50">
-      <div class="col">
-        <h2>Supported frameworks and toolchains</h2>
-      </div>
-      <div class="col">
-        <div class="p-section"
-          <p>From development environments to mission critical production systems, millions of developers trust Ubuntu as the base OS for their software projects. Discover the security, compliance, developer experience and automation features available on Ubuntu.</p>
+    <div class="p-strip is-shallow">
```

Comment:
```suggestion
    <div class="p-section--shallow">
```

Reply:
there is already a p-section dev that wraps this strip, so using strip is better in this case.